README.md

```diff
@@ -1,61 +1,97 @@
 DirectPV
 ----------
 
-![build](https://github.com/minio/directpv/workflows/Go/badge.svg) ![license](https://img.shields.io/badge/license-AGPL%20V3-blue) 
-
-DirectPV is a [CSI driver](https://kubernetes-csi.github.io/docs/introduction.html) for dynamically provisioning direct persistent volumes. 
+DirectPV is a CSI driver for [Direct Attached Storage](https://en.wikipedia.org/wiki/Direct-attached_storage). In a simpler sense, it is a distributed persistent volume manager, and not a storage system like SAN or NAS. It is useful to *discover, format, mount, schedule and monitor* drives across servers. Since Kubernetes `hostPath` and `local` PV are statically provisioned and limited in functionality, DirectPV was created to address this limitation. 
 
 ![Architecture Diagram](./architecture.png)
 
-Modern distributed datastores such as MinIO, Elastic, Cassandra, and MongoDB are designed to for locally attached drives (Local PV). These data services handle high availability and data durability by themselves. Running them on traditional SAN or NAS based CSI drivers (Network PV) adds yet another layer of replication/erasure coding and extra network hops in the datapath. This additional layer of disaggregation results in increased-complexity and poor performance.
+Distributed data stores such as object storage, databases and message queues are designed for direct attached storage, and they handle high availability and data durability by themselves. Running them on traditional SAN or NAS based CSI drivers (Network PV) adds yet another layer of replication/erasure coding and extra network hops in the data path. This additional layer of disaggregation results in increased-complexity and poor performance.
+
+### Architecture
+
+DirectPV is designed to be lightweight and scalable to 10s of 1000s of drives. It is made up of three components:
+
+- Controller
+- Node Driver
+- UI
+
+![DirectPV Architecture](./directpv_architecture.png)
```

Comment:
copy this image to `docs/screenshots` and replace with the complete path

Reply:
we can call it `docs/images`, since they are not really screenshots?